workflow/artifacts/artifactory/artifactory.go

```diff
@@ -0,0 +1,31 @@
+package artifactory
+
+import (
+wfv1 "github.com/argoproj/argo/api/workflow/v1alpha1"
+"github.com/argoproj/argo/workflow/common"
+	"fmt"
+)
+
+// HTTPArtifactDriver is the artifact driver for a HTTP URL
+type ArtifactoryArtifactDriver struct{
+Username string
```

Comment:
Is there a missing indentation for these 2 lines or is this a Github UI problem?